bindings/node/README.md

```diff
@@ -165,6 +195,24 @@ the underlying C++ Bindings.
 ## ClientEncryption
 The public interface for explicit client side encryption
 
+**NOTE:** This class must be instantiated directly from `mongodb-client-encryption`.
+To do this, you must "inject" the `mongodb` package into `mongodb-client-encryption`:
+
+```js
+// Simply importing mongodb-client-encryption returns
```

Comment:
I feel like this documentation is a bit obtuse. Why not just give them an example of using it:

```
const mongodb = require('mongodb');
const mdbce = require('mongodb-client-encryption')(mongodb);
const ClientEncryption = mdbce.ClientEncryption;
```

Also, I think we should update the injection to decorate the `mongodb` package, this is more in line with a future where we have something like `mongodb.use(require('client-encryption'))`. What about:

```
const mongodb = require('mongodb');
require('mongob-client-encryption')(mongodb);

const ClientEncryption = mongodb.ClientEncryption;
```

Reply:
I think that is a neater API. My hesitations are:

1. It makes it confusing why we need to do that for `ClientEncryption` but not `AutoEncryption`
2. Hypothetically, ESM requires that module exports be immutable. So idk the repercussions of trying to augment the exported module either when running in ESM mode or when using a transpiler that enforces this.

Reply:
How about if we introduce `mongodb.use` now then?